ingestion/src/metadata/orm_profiler/processor/orm_profiler.py

```diff
@@ -336,6 +359,168 @@ def run_table_test(
         self.log_test_result(name=test_name, result=test_case_result)
         return test_case_result
 
+    def get_test_case(
```

Comment:
maybe we want to rename this function and the one about testSuites to `get_test_case_or_create` to be a bit more explicit